### PR TITLE
remove data extensions from vscode setup

### PR DESCRIPTION
**Pull request by zzuziak**
Removing the data science packages from vscode setup - they will be included in the `data-setup` partials :pray:

### CODE REVIEW

File: WINDOWS.md [hunk 1]
```diff
@@ -490,11 +490,11 @@ The tool should start to download automatically.
 
 :warning: This extension is mandatory for you to be able to work with VS Code!
 
+
```
Comment (dmilon) [inline WINDOWS.md:493]: Delete the extra empty line here.

File: WINDOWS.md [hunk 2]
```diff
@@ -557,9 +550,9 @@ It should look like this:
 To make sure everything works fine, close VS Code, then open your Ubuntu terminal:
 
 
->\- Press `Windows` + `R`
->\- Type  `ubuntu`
->\- Press `Enter`
+- Press `Windows` + `R`
```
Comment (dmilon) [inline WINDOWS.md:553]: If we replace `>\` by normal markdown list, let's do it everywhere in every file to standardize the formatting. 

BTW, I understood why list are broken sometimes: this type of list require extra spaces at the end of each line but some text editor (my sublime for example) trim end-of-the-line spaces.

**Review by dmilon — CHANGES_REQUESTED** [review]
:+1: Just extra changes to standardize the formatting :pray:

File: WINDOWS.md (hunk 2 above)
Comment (zzuziak) [reply WINDOWS.md:553]: Yes! Exactly what happens in VSC

**Review by dmilon — APPROVED** [review]
:raised_hands:

**Pull request merged**